### ISSUE TITLE
Feature Request: Translucent/Transparent Background

### ISSUE THREAD

**Issue by gingerBill** [serge-rgb/milton#73]
Would it be possible to have a translucent/transparent background option to allow for drawing over images?

**Comment by serge-rgb** [comment]
Yup! It's definitely possible on Windows and it's in the to-do list

**Comment by steincodes** [comment]
Is it also possible on linux?

**Comment by bleachkitty** [comment]
Great program!  I've been using Mischief for a while now and this is the last feature I would need in order to switch over.  Has there been any movement on it?  Is there anything I can do to help?  (I did try to build on Window with VS 2017 but ran into a lot of dependency issues.)
